sysutils/zsh-history-substring-search/Portfile

```diff
@@ -42,3 +38,13 @@ destroot {
 }
 
 github.livecheck.regex {([^"-]+)}
+
+notes "
+To activate the history search, add this line to your configuration:
+
+    source '${prefix}/share/${name}/${name}.zsh'
+
+Please note that this line must be after loading all custom widgets,
+at the end of your .zshrc. You will need to restart your terminal for
+this change to take effect.
```

Comment:
please use `\` at the end of the lines to let MacPort wrap the lines to the terminal width.